### ISSUE TITLE
Setup second validation set consisting of small genomes that should signal overfitting

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#216]
Maybe even multiple ones.
